frontend/ui/impl/src/test/java/com/virtuslab/gitmachete/frontend/ui/impl/GitMacheteErrorReportSubmitterTest.java

```diff
@@ -20,104 +18,80 @@
-  // PowerMock API documentation Section 13 [https://javadoc.io/static/org.mockito/mockito-core/4.6.1/org/mockito/Mockito.html]
-  private static final GitMacheteErrorReportSubmitter reportSubmitter = PowerMockito
-      .spy(new GitMacheteErrorReportSubmitter());
+  private GitMacheteErrorReportSubmitter reportSubmitter;
   private static final LambdaLogger errorReportSubmitterLogger = PowerMockito.mock(LambdaLogger.class);
-  private static String defaultErrorReport;
-  private static String defaultErrorReportWithTemplate;
-  private static String errorReportWithStackTraceAndTitle;
-  private static String systemLineSeparator = "\n";
 
   @BeforeClass
-  @SneakyThrows
   public static void setUpStatic() {
-    defaultErrorReport = IOUtils.resourceToString("/error_report_uri.txt", StandardCharsets.UTF_8).strip();
-    defaultErrorReportWithTemplate = IOUtils.resourceToString("/error_report_uri_with_template.txt",
-        StandardCharsets.UTF_8).strip();
-    errorReportWithStackTraceAndTitle = IOUtils.resourceToString("/error_report_uri_with_stacktrace_title.txt",
-        StandardCharsets.UTF_8).strip();
-  }
-
-  @Before
-  public void setUp() {
     Whitebox.setInternalState(GitMacheteErrorReportSubmitter.class, errorReportSubmitterLogger);
     Whitebox.setInternalState(SystemUtils.class, "OS_NAME", "Mock OS X");
     Whitebox.setInternalState(SystemUtils.class, "OS_VERSION", "Hehe");
 
-    PowerMockito.stub(PowerMockito.method(System.class, "lineSeparator"))
-        .toReturn(systemLineSeparator);
-
     PowerMockito.mockStatic(ApplicationInfo.class);
-    PowerMockito.mockStatic(PluginId.class);
+    PowerMockito.mockStatic(PluginManagerCore.class);
+  }
+
+  @Before
+  public void setUp() {
+    reportSubmitter = new GitMacheteErrorReportSubmitter();
 
     val applicationInfo = PowerMockito.mock(ApplicationInfo.class);
     PowerMockito.when(applicationInfo.getFullApplicationName()).thenReturn("mocked IntelliJ idea");
     PowerMockito.stub(PowerMockito.method(ApplicationInfo.class, "getInstance"))
         .toReturn(applicationInfo);
-  }
 
-  private IdeaLoggingEvent[] getMockEvents(String throwableText) {
-    val customThrowable = new Throwable() {
-      @Override
-      public void printStackTrace(PrintWriter s) {
-        s.print(throwableText);
-      }
-    };
-
-    return new IdeaLoggingEvent[]{
-        new IdeaLoggingEvent(/* message */ "", customThrowable)
-    };
+    val ideaPluginDescriptor = PowerMockito.mock(IdeaPluginDescriptor.class);
+    PowerMockito.when(ideaPluginDescriptor.getVersion()).thenReturn("mock plugin version");
+    PowerMockito.stub(PowerMockito.method(PluginManagerCore.class, "getPlugin"))
+        .toReturn(ideaPluginDescriptor);
   }
 
-  private String invokeGetReportBody(IdeaLoggingEvent[] events)
-      throws Exception {
-    String reportBody = Whitebox.invokeMethod(reportSubmitter, "getReportBody", events, /* additionalInfo */ null);
-
-    Mockito
```

Comment:
Even if exception is thrown inside `getReportBody` (due to `getBugTemplate` reading from a wrong resource), we'll get to know about that from a failing test now